quarkus/service/build.gradle.kts

```diff
@@ -122,6 +123,18 @@ dependencies {
 
   testImplementation(libs.threeten.extra)
   testImplementation(libs.hawkular.agent.prometheus.scraper)
+
+  testImplementation(project(":polaris-quarkus-test-commons"))
+  testFixturesApi(enforcedPlatform(libs.quarkus.bom)) {
+    exclude(group = "org.antlr", module = "antlr4-runtime")
+    exclude(group = "org.scala-lang", module = "scala-library")
+    exclude(group = "org.scala-lang", module = "scala-reflect")
+  }
+  testFixturesApi("io.quarkus:quarkus-junit5")
+  testFixturesApi(platform(libs.testcontainers.bom))
+  testFixturesApi("org.testcontainers:testcontainers")
+  testFixturesApi("org.testcontainers:postgresql")
+  testFixturesApi("org.postgresql:postgresql")
```

Comment:
I double these test dependencies should be in the "api" scope. They are not dependencies of the API of this module's test fixtures. Would simple `testImplementation` be sufficient?

Reply:
works